src/python/WMCore/MicroService/MSPileup/MSPileupData.py

```diff
@@ -60,6 +63,14 @@ def getNewTimestamp(doc):
     return subDoc
 
 
+def customDID(pname):
+    """
+    Helper function to create custom Name from given pileup one
+    """
+    suffix = "custom-%d" % int(time.time())
```

Comment:
The longer the suffix the easier it is to fail dataset name lexicon. I would suggest to add only 3 more chars - potentially 4 - like `-V1`, `-V2`, and so on. It would require some regex and parsing though, and we might decide to simply adopt something else simpler. Bottom line is, it must be as short as possible